### DISCUSSION TITLE
windows版本使用求助   /   Windows version help

### DISCUSSION THREAD

**Discussion by 1fan-lee** [OpenRadioss/OpenRadioss#3155] [Q&A]
各位工程师们好，我在官网下载了openradioss但是不会运行脚本，我是cae方向的大学生，请问怎么正确使用GUI运行radioss求解器啊，有详细的使用步骤教程吗，官网的指南对我不太友好，我的编程基础比较弱

Hello fellow engineers, I downloaded openradioss from the official website but I don’t know how to run the script. I am a college student majoring in CAE. I would like to ask you how to use the GUI correctly to run the radioss solver. Is there a detailed tutorial on how to use it? The guide on the official website is not very useful to me. Friendly, my programming foundation is relatively weak

**Comment by sebastienVilleneuve** [comment]
Hi @1fan-lee 

您可以观看此视频，用户使用 GUI 运行 Radioss 计算。:
https://www.youtube.com/watch?v=YRwlLnXADYQ
我希望这会有所帮助。

you can watch this video, a user using the GUI to run a Radioss calculation
https://www.youtube.com/watch?v=YRwlLnXADYQ
I hope this will help.

问候,
Regards,

Sébastien

**Comment by 1fan-lee** [comment]
Dear&nbsp;Sébastien


I am so glad to receive your letters, which will help me a lot. I will read it carefully! Thanks again!


Regards,
yifan
&nbsp;
&nbsp;
------------------&nbsp;Original&nbsp;------------------
From: ***@***.***&gt;;
Date: &nbsp;Thu, Nov 21, 2024 02:11 PM
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: &nbsp;Re: [OpenRadioss/OpenRadioss] windows版本使用求助   /   Windows version help (Discussion #3155)

&nbsp;



 
Hi @1fan-lee
 
您可以观看此视频，用户使用 GUI 运行 Radioss 计算。:
 https://www.youtube.com/watch?v=YRwlLnXADYQ
 我希望这会有所帮助
 
you can watch this video, a user using the GUI to run a Radioss calculation!
 https://www.youtube.com/watch?v=YRwlLnXADYQ
 I hop this will help.
 
问候,
 Regards,
 
Sébastien
